scripts/o3de/tests/test_repo_properties.py

```diff
@@ -540,3 +541,54 @@ def mock_dryrun_print(msg, *args, **kwargs):
                             assert template in str(JSON_DICT.get('template_json_key'))
                         else:
                             assert template in expected_templates
+
+    # test copy_to_repo_subfolder
+    @pytest.mark.parametrize("repo_path, repo_name, add_gems, copy_to_repo_subfolder, \
+                            expected_result", [
+        # test copy a gem to repo subfolder
+        pytest.param(pathlib.Path('F:/repotest'), 'copytest1', [JSON_DICT['gem_archive_json_key']], temp_folder, 0)
+        
+    ])
+    def test_copy_to_repo_subfolder(self, temp_folder, repo_path, repo_name, add_gems, copy_to_repo_subfolder,
```

Comment:
is there a case where you wouldn't want `copy_to_repo_subfolder` to be true? if not you can remove it.

Reply:
Same with `repo_path`, it also seems to be changed when `copy_to_repo_subfolder` is set